### PR TITLE
fix: make caption visible if wanted

### PR DESCRIPTION
**Pull request by mjmonline**

### CODE REVIEW

File: src/components/table-content-wrapper/table-content-wrapper.jsx [hunk 1]
```diff
@@ -47,7 +49,9 @@ class TableContentWrapper extends Component {
       return (
         <div className={classes.root}>
           <Table tableLayoutFixed>
-            <caption className={classes.screenReadersOnly}>{localization.caption}</caption>
+            <caption className={cn(classes.caption, 'smart-table__caption', { [classes.screenReadersOnly]: captionHidden })}>
```
Comment (KitsuneKelso) [inline src/components/table-content-wrapper/table-content-wrapper.jsx]: Why the hardcoded class `smart-table__caption`?

Comment (Dhq) [inline src/components/table-content-wrapper/table-content-wrapper.jsx]: "smart-table__caption" class should probably not always be applied here right?

Comment (mjmonline) [reply src/components/table-content-wrapper/table-content-wrapper.jsx]: Correct, should not always be there

Comment (mjmonline) [reply src/components/table-content-wrapper/table-content-wrapper.jsx]: I want to target the caption from outside the component, but it shouldn't always be there

**Pull request merged**